docs/_pages/integrations/home-assistant-integration.md

```diff
@@ -260,6 +260,33 @@ show_toolbar: true
 {% endraw %}
 ```
 
+#### Trigger AutoEmptyDock
+
+If your robot has an AutoEmptyDock, you can manually trigger it to empty the dustbin by adding the following script to Home Assistant:
+
+`/config/scripts.yaml`
+```yaml
+{% raw %}
+vacuum_trigger_autoempty:
+  alias: vacuum_trigger_autoempty
+  sequence:
+  - service: mqtt.publish
+    data:
+      topic: valetudo/robot/AutoEmptyDockManualTriggerCapability/trigger/set
+      payload: PERFORM
+  mode: single
+{% endraw %}
+```
+
+If you use the [vacuum-card](https://github.com/denysdovhan/vacuum-card) you can also add the trigger as an additional action to it via the following snippet:
```

Comment:
The vacuum-card doesn't really work properly with valetudo so I'd rather not point to it to avoid confusion

Reply:
Should the part prior to it be removed as well then?

Or maybe a warning added that the map is problematic with it?

Reply:
Yeah

Reply:
Sorry. Not sure what you mean to say yeah to. :smile: 

Removing the earlier part? Or adding a warning to it?

Reply:
Whoops sorry. Yeah to the remove the reference to the card

Reply:
I removed the vacuum-card parts and added a very simple example for a button in the single-segment part.